prusa/connect/printer/camera.py

```diff
@@ -51,6 +54,18 @@ def is_sendable(self) -> bool:
         ]
         return all(attribute is not None for attribute in required_attributes)
 
+    def is_timelapse(self) -> bool:
+        """Is this snapshot part of the layer change shots?"""
+        required_attributes = [
+            self.camera_token,
+            # TODO: M3D-7: Identify why this is not available during saving
```

Comment:
⬇️ LOW PRIORITY

Will (possibly) be fixed with [M3D-7](https://magdiel3.atlassian.net/browse/M3D-7)